main.js

```diff
@@ -1 +1,5 @@
-alert('hello there');
+var button = document.getElementById('hiBtn');
+button.onclick = ({ target }) => {
```

Comment:
This is a ES 6 way of assigning a anonymous function. Could have been written as 
```
button.onclick = function(e) {
console.log(e.target);
}
```